pkgs/applications/misc/river-tag-overlay/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchFromSourcehut, nixos, wayland, pixman, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "river-tag-overlay";
+  version = "1.0.0";
+
+  src = fetchFromSourcehut {
+    owner = "~leon_plickat";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-hLyXdLi/ldvwPJ1oQQsH5wgflQJuXu6vhYw/qdKAV9E=";
+  };
+
+  buildInputs = [ wayland pixman pkg-config ];
```

nit: can you sort this
```suggestion
  buildInputs = [ pixman pkg-config wayland ];
```